components/automate-ui/src/app/ui.component.html

```diff
@@ -24,7 +24,10 @@
   </div>
 
   <!-- App Content -->
-  <div id="app-content-wrapper" [attr.hasNotifications]="layoutFacade.hasGlobalNotifications() ? true : null" [ngStyle]="layoutFacade.getContentStyle()">
+  <div id="app-content-wrapper"
+    [attr.hasNotifications]="layoutFacade.hasGlobalNotifications() ? true : null"
+    [attr.hasUserNotifications]="layoutFacade.hasUserNotifications() ? true : null"
```

Comment:
Do these have to evaluate to null on false or would this change be sufficient?
```suggestion
    [attr.hasNotifications]="layoutFacade.hasGlobalNotifications()"
    [attr.hasUserNotifications]="layoutFacade.hasUserNotifications()"
```

Reply:
@msorens If you want to remove the attribute, you need to pass null 😕
https://stackoverflow.com/questions/36745734/how-to-add-conditional-attribute-in-angular-2

Reply:
Kind of guessed that, but was not sure off the top of my head. Thanks for confirming. 👍